src/components/date/date.js

```diff
@@ -1,7 +1,9 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import moment from 'moment';
```

Comment:
I tried to decouple moment js from the datepicker incase we ever want to change it for something else.

All date logic should be based in the date helper if possible